lang/lang-es.js

```diff
@@ -0,0 +1,560 @@
+    PERMISSION_DENIED: 'Denegó el acceso a su webcam (cámara/micrófono).',
+    Play: 'Reproducir',
+    playsessionmsg: 'Haga clic en ‘reproducir’ para comenzar la sesión.',
+    pleasewaitWhSynNewCont: 'Por favor, espere un momento. Estamos sincronizando nuevo contenido.',
+    plswaitwhile: 'Por favor espere....',
+    poll: 'Encuesta',
+    pollblank: 'La pregunta no puede dejarse en blanco',
+    pollCancel: 'Cerrar',
+    pollDel: '¿Estás seguro de eliminar esta encuesta?',
+    pollHead: 'Vota esta encuesta',
+    pollmaybeshown: 'La encuesta será publicada',
+    pollmybpublish: 'La encuesta se publicará en breve.',
+    pollresult: 'Resultado anterior',
+    ppoll: 'Publicar encuesta',
+    PQuestions: 'Preguntas de la encuesta',
+    PQuiz: 'Publicar examen',
+    Precheck: 'Precomprobación',
+    precheckStart: 'Precomprobar',
+    prechkcmplt: 'Precomprobación completa',
+    Presult: 'Resultado de la encuesta',
+    preWllBshortly: 'La presentación se compartirá en breve.',
+    Prev: 'Anterior',
+    proposedspeed: 'Velocidad propuesta',
+    PSetting: 'Configurar encuesta',
+    Publish: 'Publicar',
+    QClosed: 'Examen cerrado',
+    Question: 'Pregunta',
+    QTquestions: 'No total de preguntas',
+    QMaximumMark: 'Marca máxima',
+    QCorrectAnswers: 'Respuestas correctas',
+    QQuestionsAttempted: 'Preguntas intentadas',
+    QYouScored: 'Has anotado',
+    Quiz: 'Examen',
+    'Quiz/page': 'Examen / página',
+    Quizes: 'Exámenes',
+    quizSubmit: 'Enviar',
+    quizmayshow: 'Prepárate para una prueba',
+    quizreviewpublish: 'Revisar y publicar examen',
+    RaiseHandStdDisabled: 'Bajar la mano',
+    RaiseHandEnable: 'Levantar la mano',
+    raiseHandNotify: 'Alumno(s) levantó la mano',
+    RaiseHandStdEnabled: 'Mano levantada',
+    readonlymode: 'Solo lectura',
+    receivedVotes: 'Votos recibidos',
+    reclaim: 'Reclamar',
+    Rectangle: 'Rectángulo',
+    rejected: 'Otra aplicación en su computadora podría estar usando su cámara web. Debe cerrar las demás aplicaciones',
+    reload: 'Recargar',
+    reloadDoc: 'Recargar documento',
+    removeContext: 'Eliminar marcador',
+    replay: 'Repetir',
+    Replay: 'Repetir desde el inicio.',
+    replay_message: 'Gracias por visualizarlo.',
+    requestScreenShare: 'Solicitar compartir pantalla',
+    requestedScreenShare: 'Se solicita compartir pantalla',
+    Reset: 'Restablecer',
+    Rtime: 'Tiempo restante',
+    rusureCquiz: '<span>¿Estás seguro de cerrar este cuestionario? </span>',
+    rvtu: 'Votos recibidos/ usuarios totales',
+    reply: 'Repetir',
+    safariBrowserIssue: 'Su navegador Safari {virtualclass1} no es compatible. Es aconsejable usar Chrome y Firefox',
+    Save: 'Grabar',
+    savesession: '¿Quieres guardar la sesión actual?',
+    savesessionTechSupport: "¿Quieres guardar la sesión actual? <br /> Una vez que descargue la sesión, las actualizaciones no estarán disponibles al volver a cargarla.",
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 170. Maximum allowed is 120.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5529949)